app/workers/enqueue_bike_possibly_found_notifications_worker.rb

```diff
@@ -0,0 +1,20 @@
+class EnqueueBikePossiblyFoundNotificationsWorker < ScheduledWorker
+  prepend ScheduledWorkerRecorder
+
+  def self.frequency
+    24.hours
+  end
+
+  def perform
+    bikes_with_matches.each do |bike, match|
+      EmailBikePossiblyFoundNotificationWorker
+        .perform_async(bike.id, match.class.to_s, match.id)
+    end
+  end
+
+  def bikes_with_matches
+    internal_matches = Bike.possibly_found_with_match
```

Comment:
Would it make sense to pluck here?

Reply:
Strictly speaking we can't, because `Bike.possibly_found_with_match` returns an array of tuples of models, but we could hypothetically `pluck` the underlying attributes instead of instantiating models. I'm on the bubble about this. 

On the one hand, doing so might be 

(1) unintuitive since AR class methods typically return models, and
(2) an over-optimization at the expense of legibility, since we access a fair number of them: aside from `id`: `owner_email`, `serial_normalized`, and the class name,

but
 
(1a) unintuitiveness might ameliorated by renaming the methods (`possibly_found_ids_with_matches` e.g.)
(1b) the method returns an Array rather than an AR relation, so it's already departing from expectations
(2a) 4 attributes is arguably not a large number
(2b) We're feeding the results to workers, so the workers can query for the full AR models based on the class name and `id`

I'm comfortable refactoring if you think it makes sense to.

Reply:
Nah. Tuples are good enough. Sorry for trying to find plucks.